model/lightgbm/examples/boston_regression.py

```diff
@@ -0,0 +1,45 @@
+from sklearn.datasets import load_boston
+from sklearn.model_selection import train_test_split
+
+from dffml import Feature, Features
+from dffml.noasync import train, accuracy
+from dffml_model_lightgbm.lgbmregressor import (
+    LGBMRegressorModel,
+    LGBMRegressorModelConfig,
+)
+
+boston = load_boston()
```

Comment:
We need to use a different dataset here

Reply:
@pdxjohnny added a different dataset in the example. ready to review.